crates/linker/Cargo.toml

```diff
@@ -16,6 +16,8 @@ roc_mono = { path = "../compiler/mono" }
 roc_build = { path = "../compiler/build" }
 roc_collections = { path = "../compiler/collections" }
 roc_error_macros = { path = "../error_macros" }
+roc_load = { path = "../compiler/load" }
+roc_reporting = { path = "../reporting" }
 bumpalo = { version = "3.11.0", features = ["collections"] }
 clap = { version = "3.2.20", default-features = false, features = ["std", "color", "suggestions"] }
```

Comment:
what is clap doing here?

Reply:
Used to be a standalone executable with command line parsing. Will remove.